priv/schemas/bundle_config_schema_v4.yaml

```diff
@@ -110,7 +110,9 @@ definitions:
         additionalProperties:
           type: string
       output:
-        type: string
+        type: object
```

Comment:
This is supposed to be an example output object that the command would return, right? If so, we can have values that are things other than strings; the `additionalProperties` constraint just artificially limits things, in that case.